unittests.py

```diff
@@ -3,22 +3,33 @@
 
 import unittest
 import doctest
+
 import pynance.dummy_test
 import pynance.textimporter_test
 import pynance.dash_viz.plot_flow_test
+import pynance.parser_test
+
 
 def doc_test_suite():
-    "Returns the testsuite doctests for all modules. Please don't forget to add new modules here."
+    """
+    Returns the testsuite doctests for all modules.
+    Please don't forget to add new modules here.
+    """
 
     import pkgutil
     import importlib
     import pynance
 
     doctest_suite = unittest.TestSuite()
+
     def add_doctests_for_module(package):
-        "Recursively walks `package` and adds doctests for all submodules and subpackages to `doctest_suite`"
+        """
```

Comment:
I don't know the Python conventions. Is this line too long? I'd like to write code so that reformatting doesn't need to happen any more.

The problem with reformatting in PRs is that it potentially creates conflicts when merging/rebasing, so it would be nice if we don't need to reformat too often after a PR is accepted. 

Do you use an automatic style checker in VSCode?

Reply:
yes sorry. yes I have autoformatting on, whenever I save and was too lazy to undo it manually. I use pylint, pep8 and autopep8. I was thinking to add some stylechecker for documentation as well... maybe we should agree on what to use to avoid these reformatting issues.

Reply:
opened #45 for this

Reply:
do you want me to change it back?

Reply:
No, it's okay, no problem. My point was just that I think reformatting should only be done when necessary because it might produce conflicts with other branches without really being urgent. But maybe the discussion about it is more costly than the benefit ;)